drivers/serial/usart_gd32.c

```diff
@@ -8,6 +8,7 @@
 #include <errno.h>
 #include <drivers/pinctrl.h>
 #include <drivers/uart.h>
+#include <soc.h>
```

Comment:
is this required?

Reply:
I think it is required because we are using `<MCUserie>_libopt.h` file. We can eliminate after rework all series.

Reply:
gd32f3x0 doesn't have MPU.

Reply:
Looks MPU introduce the `soc.h` into kernel headers.